### ISSUE TITLE
Hydrator Closes Immediately After Start

### ISSUE THREAD

**Issue by Tipphead** [DocNow/hydrator#75]
Opens fine.  Closes after I press start.  I've tried changing directories for the id files.  I have been using the program fine for a couple weeks now and have hydrated millions of tweets until now.

**Comment by edsu** [comment]
What operating system are you using?

**Comment by Tipphead** [comment]
Windows 10.  I also had VMware machines running Windows 10 and they began doing the same thing.  This is happening with no VMs open, just Hydrator on my main machine.

**Comment by edsu** [comment]
Are you able to figure out how to start it from the windows command line?

**Comment by edsu** [comment]
Also, have you tried relinking your account?

**Comment by Tipphead** [comment]
No I’ll relink my account first and update you.

**Comment by edsu** [comment]
Also, have you been running multiple instances of the hydrator in parallel?

**Comment by Tipphead** [comment]
I did try but it didn’t work so I used VMs instead.

**Comment by edsu** [comment]
And were you linking with the same account or different ones? The reason I ask is that Twitter may watch for lots of API traffic from different accounts on the same IP address.

**Comment by Tipphead** [comment]
I was using different accounts.  I did try a few times with same accounts, but I was never able to hydrate any tweets and that was over a week ago.

**Comment by edsu** [comment]
Yeah, if you use the same account they will compete with each other. And if you use different ones you run the risk of being flagged by Twitter as abusing the API. Not that you were, but that's just one of the things they might look for.

**Comment by edsu** [comment]
Still Hydrator shouldn't die like that. So if you re-linking doesn't fix it I'd love to see what gets printed to the console if you can start it at the command line.

**Comment by Tipphead** [comment]
I can’t figure out how to unlink so I’ll try command line.  I haven’t been using windows very long.

**Comment by edsu** [comment]
To relink go to Settings and click on the Link button. It should prompt you to log in at Twitter in your browser. But I'd actually really like to see the console output if you are able to start it at the command line. You should be able to locate the .exe and type in the full path to it and hit Enter. Then if you start your job you may see an error pop up?

**Comment by Tipphead** [comment]
PS C:\Users\j> C:\Users\j\AppData\Local\Programs\hydrator\Hydrator.exe                                                  PS C:\Users\j>                                                                                                          storage location: C:\Users\j\AppData\Roaming\hydrator\storage                                                           (electron) The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397          reading from X:\Twitter_Project\Datasets\Tweets_to_Trump\to_Trump_20200321_ids\to_realdonaldtrump_20200321_ids2.txt for 5a17c2c5-a2d8-4b99-85a7-9745dd473d37                                                                                    writing to X:\Twitter_Project\Datasets\Tweets_to_Trump\to_Trump_20200321_ids\to_realdonaldtrump_20200321_ids2_hydrated for 5a17c2c5-a2d8-4b99-85a7-9745dd473d37 


GUI popped up, pressed start, and it closed the GUI but command line still shows that it is running

**Comment by Tipphead** [comment]
In settings I have:
-Add an account
-Add a Microsoft account
-Add a work or school account
-the two accounts that I have with Manage next to them

I've clicked manage before which brings you to microsoft webpage, but i can't figure out how to unlink

**Comment by edsu** [comment]
Interesting, so it's still running in the terminal after the window closed? 

To relink open the Hydrator and click on Settings and then Link.

**Comment by Tipphead** [comment]
ohhhh you mean Hydrator?  okay let me try that.

**Comment by Tipphead** [comment]
I tried relinking and switching accounts with no luck.  My accounts might be flagged, but I haven't received any e-mails.  I can get a 3rd account running on a VM though.  One of my VMs wasn't working earlier.

**Comment by edsu** [comment]
Do you have a VPN? Or the ability to jump on the Internet at a different location?

**Comment by Tipphead** [comment]
I am actually at a different location right now.  Although I have run Hydrator here before.  90% of my work is done at home though.

**Comment by edsu** [comment]
And the problem persists at home too?

**Comment by Tipphead** [comment]
correct

**Comment by edsu** [comment]
The hydrator should report an error and not simply close. I think this is a duplicate of #65 so please subscribe to that ticket if you would like to get updates. It's difficult for me to test without being able to reproduce the problem :-(

**Comment by Tipphead** [comment]
Now that you say that, I have had javascript errors, but not right now.

**Comment by edsu** [comment]
Please add any you notice to the other ticket. I'm sorry I can't be of more help right now.
